src/acl/Gadgets.h

```diff
@@ -62,5 +62,18 @@ void dump_acl_access(StoreEntry * entry, const char *name, acl_access * head);
 /// \ingroup ACLAPI
 void dump_acl_list(StoreEntry * entry, ACLList * head);
 
+namespace Acl {
+
+/// let FindByName() match the given acl
+void MakeDiscoverableByName(ACL *);
+
+/// a named ACL, previously registered by MakeDiscoverableByName() (or nil)
+ACL *FindByName(const char *name);
```

Comment:
```suggestion
ACL *FindByName(const char *);
```
since the function name itself implies what the parameter means.